src/adapter/adapterDiscovery.ts

```diff
@@ -0,0 +1,486 @@
+        },
+        {
+            // Sonoff ZBDongle-E V2
+            vendorId: '1a86',
+            productId: '55d4',
+            manufacturer: 'ITEAD',
+            // /dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20240122184111-if00
+            // /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_186ff44314e2ed11b891eb5162c61111-if00-port0
+            pathRegex: '.*sonoff.*plus.*',
+        },
+        // {
+        //     // TODO: Z-station by z-wave.me (EFR32MG21A020F1024IM32)
+        //     vendorId: '',
+        //     productId: '',
+        //     // manufacturer: '',
+        //     // /dev/serial/by-id/usb-Silicon_Labs_CP2105_Dual_USB_to_UART_Bridge_Controller_012BA111-if01-port0
+        //     pathRegex: '.*CP2105.*',
+        // },
+    ],
+    zstack: [
+        {
+            // ZZH
+            vendorId: '0403',
+            productId: '6015',
+            manufacturer: 'Electrolama',
+            pathRegex: '.*electrolame.*', // TODO
+        },
+        {
+            // slae.sh cc2652rb
+            vendorId: '10c4',
+            productId: 'ea60',
+            // manufacturer: '',
+            pathRegex: '.*2652.*',
+        },
+        {
+            // Sonoff ZBDongle-P (CC2652P)
+            vendorId: '10c4',
+            productId: 'ea60',
+            manufacturer: 'ITEAD',
+            // /dev/serial/by-id/usb-Silicon_Labs_Sonoff_Zigbee_3.0_USB_Dongle_Plus_0111-if00-port0
+            // /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_b8b49abd27a6ed11a280eba32981d111-if00-port0
+            pathRegex: '.*sonoff.*plus.*',
+        },
+        {
+            // CC2538
+            vendorId: '0451',
+            productId: '16c8',
+            manufacturer: 'Texas Instruments',
+            // zStack30x: /dev/serial/by-id/usb-Texas_Instruments_CC2538_USB_CDC-if00
+            pathRegex: '.*CC2538.*',
+        },
+        {
+            // CC2531
+            vendorId: '0451',
+            productId: '16a8',
+            manufacturer: 'Texas Instruments',
+            // /dev/serial/by-id/usb-Texas_Instruments_TI_CC2531_USB_CDC___0X00124B0018ED1111-if00
+            pathRegex: '.*CC2531.*',
+        },
+        {
+            // CC1352P_2
+            vendorId: '0451',
+            productId: 'bef3',
+            manufacturer: 'Texas Instruments',
+            pathRegex: '.*CC1352P_2.*', // TODO
+        },
+        {
+            // CC26X2R1
+            vendorId: '0451',
+            productId: 'bef3',
+            manufacturer: 'Texas Instruments',
+            pathRegex: '.*CC26X2R1.*', // TODO
+        },
+        {
+            // SMLight slzb-07p7
+            vendorId: '10c4',
+            productId: 'ea60',
+            manufacturer: 'SMLIGHT',
+            // /dev/serial/by-id/usb-SMLIGHT_SMLIGHT_SLZB-07p7_be9faa0786e1ea11bd68dc2d9a583111-if00-port0
+            pathRegex: '.*SLZB-07p7.*',
+        },
+        {
+            // TubesZB ?
```

Comment:
@tube0013 Can you take a look at the TubesZB fingerprints and confirm the data? If you can also provide the fingerprints for your other devices, that'd be great.

Reply:
I'm going to be adopting the generic zeroconf/mdns discovery coming to HA/ZHA: https://github.com/home-assistant/core/pull/126294

Reply:
These are for USB. You don't have any USB device? I found a couple of fingerprints in zigpy code, but maybe they are no longer relevant?

Reply:
my bad I should read, this info is still valid: https://github.com/home-assistant/core/blob/dcb6c9a13375d5db73211f250a52c5342d36307d/homeassistant/components/zha/manifest.json#L51 

through L51.

Reply:
Did you set the `manufacturer` property as well? (that would add an extra bit of info to match against).

Reply:
I set product string to "TubesZB" and serial is set as 971207DO or 971229NY

At least on the ch340b I'm not aware how to write a manufacturer string.